syntax = "proto2";

message ExerciseClientData {
  optional double total_distance = 1;
  optional int32 curr_heart_rate = 2;
  optional double avg_speed = 3;
  optional int64 curr_time = 4;
  optional int32 exercise_duration = 5;
}